site/docs/components/progress-bar.mdx

```diff
@@ -10,12 +10,13 @@ needToKnow:
 health:
   designed: true
   documented: true
-  implemented: false
-  latestDesign: false
-  allVariants: false
+  implemented: true
+  latestDesign: true
+  allVariants: true
   responsive: false
   internationalized: false
   accessible: false
```

Comment:
I _think_  it is accessible due to the aria attributes I added (which are also recommended in https://cultureamp.design/components/progress-bar/ ), but perhaps this is best left to false until it has been officially audited.

Reply:
For now we are just doing our best to assess this ourselves, and we will probably introduce some data for whether these have been part of an audit or not in the future.

I can take a look tomorrow if you like and see if I can find any accessibility issues.

Reply:
Unfortunately I'm pretty sure most of the colours are going to fail contrast ratio requirements. I'm not sure if the label is enough to invalidate that or not, but I guess we'll have this as `false` for now.